streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamThread.java

```diff
@@ -268,17 +274,19 @@ public void onPartitionsAssigned(final Collection<TopicPartition> assignment) {
             final long start = time.milliseconds();
             try {
                 if (streamThread.setState(State.PARTITIONS_ASSIGNED) == null) {
-                    return;
-                }
-                if (streamThread.assignmentErrorCode.get() == StreamsPartitionAssignor.Error.NONE.code()) {
+                    log.debug("Skipping task creation in rebalance because we are already in {} state.",
```

Comment:
Shouldn't the indentation be as follows?
```
log.debug(
    "Skipping task creation in rebalance because we are already in {} state.",
    streamThread.state()
);
```